lib/datadog/appsec/assets/blocked.text

```diff
@@ -0,0 +1,5 @@
+You've been blocked
```

Comment:
These `assets/blocked*` responses don't have any information about the host service or website, meaning it looks like the page is comes from Datadog servers ("Security provided by Datadog") instead of from, say, "My Bank" if you end up being blocked while browing a `mybank.zbc` page.

Is branding, or at least a simple link saying "Go back to the homepage: mybank.zbc", something we are concern with for these AppSec responses?

Reply:
This is a first implementation with a static page, that has been defined in an internal RFC.

The very same concerns have been raised on the aforementioned document, and there are improvements coming up but they are not formalized yet.